pipelines/jobs/configurations/jdk11_pipeline_config.groovy

```diff
@@ -45,14 +42,14 @@ class Config11 {
                 buildArgs : [
                         hotspot : '--jvm-variant client,server'
                 ],
-                test                : ['sanity.openjdk', 'sanity.perf', 'sanity.system', 'extended.system']
+                test                : false
         ],
 
         x64WindowsXL    : [
                 os                   : 'windows',
                 arch                 : 'x64',
                 additionalNodeLabels : 'win2012&&vs2017',
-                test                 : ['sanity.openjdk', 'sanity.system'],
+                test                 : false
```

Comment:
Nees a ','

Reply:
Added - thanks :-) If anything was likely to be wrong with this it was going to be that - I even tried to double check them